docs/account-management/permissions/index.md

```diff
@@ -19,6 +19,13 @@ Click the name of the GeoFabric to expand it and view permissions for specific f
 - **No Access -** No access to the feature and related files.
 - **Use Default -** Uses default settings as specified on the bottom row.
 
-You can configure permissions at a more granular level by assigning permissions at the database, collection, or stream level. Each level of permissions has its own default. You can change default permissions for all levels except database, which permanently defaults to **No Access**. You must individually assign **Access** or **Administrate** database permissions to users or API keys.
+You can configure permissions at a more granular level by assigning permissions at the following levels:
 
-When you set permissions to the database or collection level, the lower levels (collection and stream) inherit the highest level of access. For example, if you set database permissions to **Access** and collection permissions to **Administrate**, then stream permissions automatically inherit the **Administrate** access level because the collection-level permission is higher than the database-level permission.
+- Fabric
+- Collections
+- Streams
+- Query Workers
+
+Each level of permissions has its own default. You can change default permissions for all levels except fabric, which permanently defaults to **No Access**. You must individually assign **Access** or **Administrate** fabric permissions to users or API keys.
+
+When you set permissions to the fabric or collection level, the lower levels (stream and query worker) inherit the highest level of access. For example, if you set database permissions to **Access** and collection permissions to **Administrate**, then stream permissions automatically inherit the **Administrate** access level because the collection-level permission is higher than the database-level permission.
```

Comment:
The fabric level permissions are only at higher level and rest all (collections, streams, queryWorkers) are at lower level. There's no relation/inheritance of permissions amongst collections, streams, queryWorkers permissions.

Reply:
Updated. How's this?

Reply:
LGTM